traffic_ops/bin/traffic_ops_ort.pl

```diff
@@ -1619,10 +1619,12 @@ sub get_cookie {
 		&sleep_rand($login_dispersion);
 	}
 
-	my $url = $to_host . "/login";
-	my $response = $lwp_conn->post( $url, [ 'u' => $u, 'p' => $p ], %headers );
+	my $url = $to_host . "/api/1.3/user/login";
+    my $json = '{"u" : "' . $u . '", "p" : "' . $p . '"}';
```

Comment:
would be more readable using the qq operator:
```
my $json = qq/{ "u": "$u", "p": "$p"}/;
```

Reply:
thanks.  fixed.